wellies/deployment.py

```diff
@@ -91,6 +91,7 @@ def deploy_suite(
     name: str,
     hostname: str,
     deploy_dir: str,
+    deploy_user: str = None,
```

Comment:
This already exists. The "deploy_user" is "user" in the context of the deployment function, but defined as "deploy_user" in the yaml configuration.